src/main/java/Duke.java

```diff
@@ -3,119 +3,43 @@
-                    storage.writeFile(lst);
-                } else if (currTask.equals("event")) {
-                    int cutoff = 0;
-                    String task = "";
-                    String time = "";
-                    for (int j = 1; j < myArray.length; j++) {
-                        if (myArray[j].charAt(0) == '/') {
-                            cutoff = j;
-                            break;
-                        } else {
-                            if (j != 1) {
-                                task += " ";
-                            }
-                            task += myArray[j];
-                            cutoff = j;
-                        }
-                    }
-                    for (int k = cutoff + 1; k < myArray.length; k++) {
-                        if (k != cutoff + 1) {
-                            time += " ";
-                        }
-                        time += myArray[k];
-                    }
-                    lst.add(new Event(task, time));
-                    counter++;
-                    System.out.println("Got it. I've added this task:\n  " + lst.get(counter - 1)
-                            + "\nNow you have " + counter + " tasks in the list.");
-                    storage.writeFile(lst);
-                } else if (currTask.equals("todo")) {
-                    String task = "";
-                    for (int j = 1; j < myArray.length; j++) {
-                        if (j != 1) {
-                            task += " ";
-                        }
-                        task += myArray[j];
-                    }
-                    lst.add(new ToDo(task));
-                    counter++;
-                    System.out.println("Got it. I've added this task:\n  " + lst.get(counter - 1)
-                            + "\nNow you have " + counter + " tasks in the list.");
-                    storage.writeFile(lst);
-                } else if (currTask.equals("delete")) {
-                    String numString = myArray[1];
-                    int num = Integer.valueOf(numString);
-                    Task curr = lst.remove(num - 1);
-                    counter--;
-                    System.out.println("Noted. I've removed this task:\n  " + curr
-                            + "\nNow you have " + counter + " tasks in the list.");
-                    storage.writeFile(lst);
-                } else {
-                    throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
-                }
-            }
-            storage.writeFile(lst);
+            tasks = storage.readFile();
         } catch (DukeException e) {
-            System.out.println(e.getMessage());
-        } catch (IndexOutOfBoundsException e) {
-                System.out.println("Invalid item in the list!");
-        } catch (IOException e) {
-            System.out.println("Something went wrong: " + e.getMessage());
```

Comment:
Your OOP is very well done! It seems like you managed to clear up a lot of lines and it looks really neat now.